src/tcp_wrap.cc

```diff
@@ -335,6 +335,18 @@ Local<Object> AddressToJS(Environment* env,
   case AF_INET6:
     a6 = reinterpret_cast<const sockaddr_in6*>(addr);
     uv_inet_ntop(AF_INET6, &a6->sin6_addr, ip, sizeof ip);
+    // Add an interface identifier to a link local address
```

Comment:
Can you punctuate comments here and in the test?

Reply:
Sorry but I don't understand what you mean with punctuation here.

Reply:
Dot at the end. :-)